Update README.md (#9)
adnovum/gong-notifier: gong-notifier-email/README.md

GONG_EMAIL_EVENTS = deploy.failed
```

> If you're interested in events only for a specific stage, use the `all` keyword to receive the default events from that.

Reviewer: ```suggestion
If you're interested in getting all **default** events but only for a specific stage, use `all` instead of a specific event:
```
